kerl

```diff
@@ -135,6 +135,7 @@ KERL_GIT_DIR=${KERL_GIT_DIR:="${KERL_BASE_DIR:?}"/gits}
 KERL_GIT_BASE=https://raw.githubusercontent.com/kerl/kerl/master
 KERL_COLORIZE=${KERL_COLORIZE:=$KERL_COLOR_AVAILABLE}
 KERL_INCLUDE_RELEASE_CANDIDATES=${KERL_INCLUDE_RELEASE_CANDIDATES:=no}
+KERL_CHECK_BUILD_PACKAGES=${KERL_CHECK_BUILD_PACKAGES:="1"}
```

Comment:
Behaviour is kept, or do you prefer to change the default?

Reply:
I'd prefer if the default were "yes" or "true" rather than 1

Reply:
I'll change it. 👍 I did think about it, at a point in time, but while looking at the other variables, found that `KERL_COLORIZE`, and `KERL_AUTOCLEAN` both mention `0`/`1`, and not `no`/`yes`.

Reply:
Granted `KERL_INCLUDE_RELEASE_CANDIDATES` has `no` as a default... 😄

Reply:
Granted `KERL_INCLUDE_RELEASE_CANDIDATES` has `no` as a default... 😄

Reply:
6505b69